crates/node-builder/src/rpc.rs

```diff
@@ -300,7 +300,8 @@ where
 
     let launch_auth = auth_module.clone().start_server(auth_config).map_ok(|handle| {
         let addr = handle.local_addr();
-        info!(target: "reth::cli", url=%addr, "RPC auth server started");
+        let ipc_endpoint = handle.ipc_enpodint();
+        info!(target: "reth::cli", url=%addr, ipc_endpoint=%ipc_endpoint,"RPC auth server started");
```

Comment:
I'd like to have two logs for this, one for HTTP and __if__ we have ipc, also for ipc